docs/guide/breakpoint-preview.md

```diff
@@ -0,0 +1,159 @@
+          icon: 'tablet-icon',
+          shortcut: true
+        },
+        mobile: {
+          label: 'Mobile',
+          width: '414px',
+          height: '896px',
+          icon: 'cellphone-icon',
+          shortcut: true
+        },
+        ultrawide: {
+          label: 'Ultrawide Monitor',
+          width: '2560px',
+          height: '1440px'
+        },
+        laptopLarge: {
+          label: 'Large Laptop',
+          width: '1680px',
+          height: '1050px'
+        },
+        ipadPro: {
+          label: 'iPad Pro',
+          width: '1366px',
+          height: '1024px'
+        },
+        surfaceDuo: {
+          label: 'Surface Duo',
+          width: '540px',
+          height: '720px'
+        },
+        galaxyFold: {
+          label: 'Galaxy Fold',
+          width: '280px',
+          height: '653px'
+        }
+      }
+    }
+  }
+};
+```
+<template v-slot:caption>
+modules/@apostrophecms/asset/index.js
+</template>
+</AposCodeBlock>
+
+You can learn about all the properties that can be passed to the `breakpointPreviewMode` option on the [`@apostrophecms/asset` reference page](/reference/modules/asset.md#breakpointPreviewMode). The two most critical are `enable` and `screens`. If you pass `enable: false` it will remove the breakpoint preview icons and menu from the admin-bar. The `screens` property takes an object where each property is a different breakpoint. Adding the `breakpointPreviewMode` at the project level will override any default values, so you need to pass `enable: true` if you wish to add additional brekpoints.
+
+Each of the individual breakpoint properties in the `screen` object take a `label` that will be displayed to the user when they hover over the icon. The `icon` property supplies the icon that will be displayed if the breakpoint is also given a property of `shortcut: true`. This icon name should be either an icon you can [register in this module](/reference/module-api/module-options.md#icon) or an icon that is [already registered](/reference/module-api/module-overview.md#icons). Finally, each breakpoint needs the `width` and `height` of the container that corresponds to the device being emulated.
+
+---
+
+### Limitations and Differences
+While the ApostropheCMS breakpoint Preview effectively converts many media queries to container queries, there are some key limitations and differences to be aware of. These limitations stem from fundamental differences between **media queries** (which target the viewport) and **container queries** (which target individual containers).
+
+#### 1. **Unsupported Parameters**
+Certain media query conditions are not supported or do not translate well to container queries:
+- **Height-based queries**:
+  - `min-height` and `max-height` conditions cannot be applied to containers. Container queries only consider the container's width.
+  - Example of unsupported query:
+    ```css
+    @media (min-height: 600px) { ... }
+    ```
+- **Orientation queries**:
```

Comment:
Huh I think we can probably fix this one but that's not a PR comment, it's just an observation.

Reply:
Yeah, I'm not sure why this is a limitation. It isn't a hard fix (I don't think), just a conversion from `min-height` to `height >=`